frontend/public/components/dashboard/details-card/detail-item.tsx

```diff
@@ -1,20 +1,25 @@
 import * as React from 'react';
+import * as classNames from 'classnames';
 import { LoadingInline } from '../../utils';
 
 export const DetailItem: React.FC<DetailItemProps> = React.memo(
   ({ title, isLoading = false, children, error = false }) => {
     let status: React.ReactNode;
+    let ddClass = 'co-details-card__item-value';
     if (error) {
       status = <span className="text-secondary">Not available</span>;
     } else if (isLoading) {
       status = <LoadingInline />;
     } else {
       status = children;
+      if (typeof status === 'string') {
```

Comment:
We don't want to put this on all strings. `className` should be an optional prop and let the caller decide.

Reply:
No problem with moving it one level up, but my thinking is that this should be consistent.
When looking at other Detail Cards, the textual values rendered there can be expected to be copied by the user. It refers to values like versions, URL, platform, channel, names of resources or ip addresses. 

@andybraren , wdyt?

Reply:
Good question. Maybe it's just my Friday brain, but I've gone back and forth on this a couple of times now. 😄

If we don't enforce `co-select-to-copy` on every non-blue string in every Details card for consistency, I can easily imagine users getting tripped up when they try to copy `stable-4.2` (for example) and it doesn't behave like other similar-looking strings because we forgot a prop or didn't expect them to want to copy it.

But if we do enforce it, I can imagine it being slightly weird when other fields like Date Created (with the globe icon) can be quick-copied while in a Details card, but not in other places it may show up in the Console.

I guess from a code perspective it probably makes sense to make `className` an optional prop as @spadgett said to avoid potential future conflicts, but for now we'd apply `co-select-to-copy` to pretty much every string we've thought of so far. We can discuss/think more about this though.

Reply:
done - the decision is moved one level up to the calling code. So I have added the class to all relevant occurrences in the cluster-, pod- and vm-dashboards.

Reply:
Additional dashboards should be fixed in follow-ups. I can volunteer here, just let me know if agreed.